pkg/storage/snapshot/restore.go

```diff
@@ -667,15 +675,15 @@ func (t *vmRestoreTarget) restoreInstancetypeControllerRevision(vmSnapshotRevisi
 
 func (t *vmRestoreTarget) restoreInstancetypeControllerRevisions(vm *kubevirtv1.VirtualMachine) error {
 	if vm.Spec.Instancetype != nil && vm.Spec.Instancetype.RevisionName != "" {
-		restoredCR, err := t.restoreInstancetypeControllerRevision(vm.Spec.Instancetype.RevisionName, t.vmRestore.Spec.VirtualMachineSnapshotName, vm)
+		restoredCR, err := t.restoreInstancetypeControllerRevision(vm.Spec.Instancetype.RevisionName, t.vmRestore.Spec.VirtualMachineSnapshotName, vm, false)
```

Comment:
To be honest I'm not a fan of passing the `isPreference` bool through the whole call stack. But maybe that's something for a cleanup PR.

Reply:
ACK yeah I can look in a follow up.

Reply:
It could be a use case for generics.